### ISSUE TITLE
Настраивается ли проект для слежения за изменениями в файлах и автоматической перезагрузки страницы как например watch в Gulp?

### ISSUE THREAD

**Issue by vasal886** [bem-site/bem-forum-content-ru#1155]

**Comment by Degtyarev-vg** [comment]
Да, настраивается, вот тут https://ru.bem.info/forum/1138/ в комментариях всё предельно ясно описано. Но это просто как вариант, можно и по другому всё это сделать.

**Comment by dima117** [comment]
Еще посмотрите мини-сервер для локальной БЭМ-разработки https://www.npmjs.com/package/direct-dev-server, он тоже пересобирает файлы при изменении.
